test/Conversion/TorchToTosa/basic.mlir

```diff
@@ -1134,3 +1134,51 @@ func.func @torch.aten.remainder.Scalar(%arg0: !torch.vtensor<[2, 4],f32>) -> !to
   %0 = torch.aten.remainder.Scalar %arg0, %int2 : !torch.vtensor<[2, 4],f32>, !torch.int -> !torch.vtensor<[2, 4],f32>
   return %0 : !torch.vtensor<[2, 4],f32>
 }
+
```

Comment:
@mgehre-amd , I guess the normal flow already generated `aten.gelu`. Would you say we don't need this test?

Reply:
I suggest to revert the changes to this file. The normal gelu lowering should be tested via end2end tests already.